libs/web/components/src/Dialog.tsx

```diff
@@ -0,0 +1,58 @@
+import { PropsWithChildren } from "@app-gov/web/types";
+import { Dialog as BaseDialog, Transition } from "@headlessui/react";
+import { FC, MutableRefObject, Fragment } from "react";
```

Comment:
Look at that, `headlessui` was the other library I was thinking of - their `dropdown` component as our `Select`

Reply:
Yeah I actually prefer the Daisy way of handling the Select, uses the native dropdown. Will consider using Dropdown for this